net-dns/knot/knot-2.0.2.ebuild

```diff
@@ -88,6 +88,10 @@ src_install() {
 	fi
 
 	newinitd "${FILESDIR}/knot.init" knot
+
+	if use systemd; then
```

Comment:
That's nice to know, however I don't know git well enough to edit the pull request.